Integrated-Windows-Auth-Flow/src/main/java/TokenCacheAspect.java

```diff
@@ -26,31 +20,5 @@ public void beforeCacheAccess(ITokenCacheAccessContext iTokenCacheAccessContext)
     @Override
     public void afterCacheAccess(ITokenCacheAccessContext iTokenCacheAccessContext) {
         data = iTokenCacheAccessContext.tokenCache().serialize();
-        // you could implement logic here to write changes to file
-    }
-
-    private static String readDataFromFile(String resource) {
```

Comment:
Instead of getting rid of readDataFromFile, we should discuss on what a good persistence implementation looks like. @SomkaPe had some ideas on how to improve this. Perhaps we could even use the extensions library.